go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/gorilla/websocket v1.5.0
 	github.com/hashicorp/go-getter v1.6.2
 	github.com/hashicorp/go-version v1.6.0
-	github.com/hashicorp/terraform-config-inspect v0.0.0-20200806211835-c481b8bfa41e
+	github.com/hashicorp/terraform-config-inspect v0.0.0-20221012204812-413b69327090
```

Comment:
could you update to `v0.0.0-20221020162138-81db043ad408` please?

Reply:
Done